components/service/telemetry/src/main/java/org/mozilla/telemetry/storage/FileTelemetryStorage.java

```diff
@@ -99,6 +99,10 @@ private void storePing(TelemetryPing ping) {
 
         final String serializedPing = serializer.serialize(ping);
 
+        if (configuration.isDebugMode()) {
+            logger.warn(serializedPing, null);
```

Comment:
We should use logger.debug here. It's not a warning, just debugging information :).